subprojects/performance/src/templates/cpp-multi-project-root/settings.gradle

```diff
@@ -0,0 +1,3 @@
+rootProject.name = 'root'
```

Comment:
I think this is actually unnecessary because the project generator generates the `settings.gradle` from `subprojects/internal-performance-testing/src/templates/settings.gradle`.  I tried removing this from the set of root templates and the build was generated correctly still.

Reply:
Removed